docs/articles/clustering/cluster-overview.md

```diff
@@ -236,6 +236,7 @@ A node might also exit the cluster gracefully, preventing it from being marked a
 <iframe width="560" height="315" src="https://www.youtube.com/embed/mUTKvGyxbOA" frameborder="0" allowfullscreen></iframe>
 <!-- markdownlint-enable MD033 -->
 
+* [Cluster Member Roles](member-roles.md)
```

Comment:
use `xref`

Reply:
Also, should probably add a comment earlier in the actual literature of this page about cluster roles, rather than just slapping a link in the "additional resources" section - introduce the context of what a role is sooner while we're introducing other Akka.Cluster concepts.